### PR TITLE
add forgot password FAQ

### PR DESCRIPTION
**Pull request by conversayShawn**

### CODE REVIEW

File: docs/faq/questions/cloud-faq.mdx [hunk 1]
```diff
@@ -55,6 +55,22 @@ Cloud.
...
+## <Icon name="angle-right" /> What if I can't access my Cypress Cloud account?
+
+1. If you haven't signed up, see the
+   [Sign Up with an Invitation](/guides/cloud/users#Sign-Up-with-an-Invitation)
+   instructions
+
+2. Try resetting your password. If you never received the email, try the
+   following:
+
+- Try signing in with Google or GitHub
```
Comment (jaffrepaul) [inline docs/faq/questions/cloud-faq.mdx:67]: Let's tab these bullets in one space. Makes it a but easier to read with the hierarchy.

File: docs/faq/questions/cloud-faq.mdx [hunk 2]
```diff
@@ -55,6 +55,22 @@ Cloud.
 A [user](/guides/cloud/users) is anyone with a login to Cypress Cloud who has
 been invited to see and review the test results of your organization.
 
+## <Icon name="angle-right" /> What if I can't access my Cypress Cloud account?
+
+1. If you haven't signed up, see the
+   [Sign Up with an Invitation](/guides/cloud/users#Sign-Up-with-an-Invitation)
```
Comment (jaffrepaul) [inline docs/faq/questions/cloud-faq.mdx:61]:
```suggestion
   [Sign Up with an Invitation](/guides/cloud/users#Sign-Up-with-an-Invitation).
```

File: docs/guides/cloud/users.mdx [hunk 3]
```diff
@@ -54,6 +54,11 @@ Cypress Cloud. From there, you will have the choice of authenitication type:
 Refer to our
 [Enterprise SSO Guide](https://on.cypress.io/organizations#Slack-Integration)
 
+#### **Locked Out or Forgot Password**
+
+See
+[What if I can't access my Cypress Cloud account](/faq/questions/cloud-faq#What-if-I-cant-access-my-Cypress-Cloud-account)
```
Comment (jaffrepaul) [inline docs/guides/cloud/users.mdx:60]:
```suggestion
[What if I can't access my Cypress Cloud account](/faq/questions/cloud-faq#What-if-I-cant-access-my-Cypress-Cloud-account).
```

File: docs/guides/cloud/users.mdx [hunk 4]
```diff
@@ -54,6 +54,11 @@ Cypress Cloud. From there, you will have the choice of authenitication type:
 Refer to our
 [Enterprise SSO Guide](https://on.cypress.io/organizations#Slack-Integration)
 
+#### **Locked Out or Forgot Password**
```
Comment (jaffrepaul) [inline docs/guides/cloud/users.mdx:57]: These section headers render as h4's so they don't need the additional "**" bold syntax. EAxtra high fives if you can pull them off of the previous headers as well while in here 🙏 😄

**Review by jaffrepaul — COMMENTED** [review]
Cool! Left a few small suggestions and a request for help 😄

**Comment by conversayShawn** [comment]
I rebased the branch, but let me know if I missed anything else.

File: docs/faq/questions/cloud-faq.mdx [hunk 5]
```diff
@@ -56,6 +56,22 @@ Cloud.
 A [user](/guides/cloud/users) is anyone with a login to Cypress Cloud who has
 been invited to see and review the test results of your organization.
 
+## <Icon name="angle-right" /> What if I can't access my Cypress Cloud account?
+
```
Comment (jaffrepaul) [inline docs/faq/questions/cloud-faq.mdx:60]: Weird, I posted this update a few days ago. Maybe I never saved it. My bad on this suggestion I made earlier. I didn't notice the line break with the word "instructions" on the next line. If you pull out the "." I suggested after the link? Then we should be in good shape.

**Review by jaffrepaul — APPROVED** [review]

**Pull request merged**